Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+    private static final int count = 100;

```

Comment:
```suggestion
    private static final int MAX_NUMBER = 100;

```

static final is constant. Constants in Java names in UPPER_CASE